### PR TITLE
fix(examples): update angular-instantsearch versions (#258)

### PR DESCRIPTION
**Pull request by tkrugg**
This is fixing https://github.com/algolia/angular-instantsearch/issues/258.

### CODE REVIEW

File: examples/e-commerce/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "@angular/platform-browser": "^6.0.2",
     "@angular/platform-browser-dynamic": "^6.0.2",
     "@angular/router": "^6.0.2",
-    "angular-instantsearch": "1.0.1",
+    "angular-instantsearch": "^2.0.4",
```
Comment (tkrugg) [inline examples/e-commerce/package.json]: I've kept the `^`. Any recommendation against it? @algolia/instantsearch-for-websites

Comment (samouss) [reply examples/e-commerce/package.json]: Nope not for the production dependencies.

**Comment by samouss** [comment]
Why you didn't update the remaining examples?

File: examples/e-commerce/package.json (hunk 1 above)
Comment (tkrugg) [reply examples/e-commerce/package.json]: ok thanks for explaining why offline :)

File: examples/server-side-rendering/package.json [hunk 2]
```diff
@@ -29,25 +29,25 @@
...
-    "@nguniversal/express-engine": "^5.0.0-beta.5",
-    "@nguniversal/module-map-ngfactory-loader": "^5.0.0-beta.5",
-    "angular-instantsearch": "1.0.1",
-    "core-js": "^2.4.1",
-    "instantsearch.js": "^2.3.0",
-    "nouislider": "^10.1.0",
-    "preboot": "^6.0.0-beta.0",
-    "rxjs": "^5.5.2",
-    "zone.js": "^0.8.14"
+    "@angular/animations": "5.0.0",
```
Comment (samouss) [inline examples/server-side-rendering/package.json]: Worth trying update to v6

Comment (tkrugg) [reply examples/server-side-rendering/package.json]: It [seems migration from 5 to 6 is not straight forward](https://stackoverflow.com/questions/48970553/want-to-upgrade-project-from-angular-v5-to-angular-v6#answer-49474334). There are a few changes required to the code. I'll isolate them in a another PR if you don't mind

**Pull request merged**